src/io/opendss.jl

```diff
@@ -523,7 +523,7 @@ function dss2tppm_branch!(tppm_data::Dict, dss_data::Dict, import_all::Bool)
 
             rmatrix = parse_matrix(defaults["rmatrix"], nodes, nconductors) * 3  # why?
             xmatrix = parse_matrix(defaults["xmatrix"], nodes, nconductors) * 3  # why?
-            cmatrix = parse_matrix(defaults["cmatrix"], nodes, nconductors)
+            cmatrix = parse_matrix(defaults["cmatrix"], nodes, nconductors) / 3
```

Comment:
Since we are now aware of the origin of this factor, could you update the line comments to reflect this?